src/chigame/games/views.py

```diff
@@ -196,6 +196,12 @@ class TournamentDeleteView(DeleteView):
     success_url = reverse_lazy("tournament-list")
 
 
+def TournamentChatDetailView(request, pk):
+    match = Match.objects.get(pk=pk)
```

Comment:
<img width="747" alt="image" src="https://github.com/uchicago-cs/chigame/assets/102563648/01e7240b-0bf0-4b0c-8c17-ddd7d6eeddf5">
I'm getting this error when trying to access chat. I think it's because the pk being used for match is the one from the tournament. I am on a tournament with pk=3, but the only match on this page has pk=2, and I do not have a match with pk=3. I'm a bit confused as to why you want the chat functionality only for matches right now. With the indexing and because it is being implemented into tournaments, it makes more sense to me to have a chat for the entire tournament as a whole.

Reply:
We matched chat to match as when we were defining the model for chat, we only assigned match as a foreign key to chat, as we want each chat to exclusive to each match. We can try to change the model to map toward tournament, but I think we intend to make chat mapped to match whening designing the ERD?

Reply:
Please see latest commit~